### PR TITLE
Update wechat from 2.3.25.18_1558616799 to 2.3.25.21

### PR DESCRIPTION
**Pull request by ElfSundae**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/wechat.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'wechat' do
-  version '2.3.25.18_1558616799'
-  sha256 'bab2930993fefdf1cb4bb663c4e365665d67cd96137977408049b1d1c6abe2d6'
+  version '2.3.25.21'
```
Comment (vitorgalvao) [inline Casks/wechat.rb]: Appcast shows `2.3.25.19`.

Comment (reitermarkus) [reply Casks/wechat.rb]: I don't know what's going on with this appcast. Before it was showing `2.3.25.20` for me, now it is showing `2.3.25.18`.

Comment (reitermarkus) [reply Casks/wechat.rb]: And now `2.3.25.20` again.

Comment (ElfSundae) [reply Casks/wechat.rb]: :scream: Maybe it is a network issue? For me, it has always been shown normally.

Comment (adidalal) [reply Casks/wechat.rb]: `2.3.25.18` here. They're likely doing staged/regional rollouts or some form of A/B testing

Comment (vitorgalvao) [reply Casks/wechat.rb]: Let’s go for the highest one, then. @ElfSundae, weren’t you able to find a versioned `url` for this?

Comment (ElfSundae) [reply Casks/wechat.rb]: @vitorgalvao No versioned `url` since `2.3.25.19`.

**Pull request merged**